sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdClientChannelPool.java

```diff
@@ -947,19 +1038,17 @@ private Channel pollChannel() {
             return first;  // because this.close -> this.close0 -> this.pollChannel
         }
 
-        if (this.isChannelServiceable(first)) {
+        if (this.isChannelServiceable(first, false)) {
```

Comment:
Please add comments on reason for "false' choice.

Reply:
Added comments